src/GRA.Domain.Service/PerformerSchedulingService.cs

```diff
@@ -1451,6 +1451,7 @@ private async Task RemoveKitImageAsync(PsKitImage image)
                 branchSelection.BackToBackProgram);
 
             branchSelection.KitId = null;
+            branchSelection.IsDeleted = false;
```

Comment:
Probably not necessary, bools are false by default.

Reply:
Fixed in 7ea0e75f2f09c9275a89a5a37a81ef42cf2b2fba